<launch>
  <arg name="robot" default="scitos"/>
  <arg name="num_of_views" default="500"/>
  <arg name="output_file" default="$(find viper)/data/op/$(arg robot)/views.json"/>

  <include file="$(find viper)/launch/$(arg robot)/_generate_views.launch"/>

  <node name="generate_views" pkg="viper" type="gen_views.py" output="screen">
    <param name="robot" type="string" value="$(arg robot)"/>
    <param name="num_of_views" type="int" value="$(arg num_of_views)"/>
    <param name="output_file" type="string" value="$(arg output_file)"/>
  </node>
</launch>
